### DISCUSSION TITLE
使用alist复制大体积视频到本地，下载的是被转码的版本

### DISCUSSION THREAD

**Discussion by coolyzp** [AlistGo/alist#3080] [Q&A]
原文件Braveheart.1995.2160p.BluRay.REMUX.HEVC.DTS-HD.MA.TrueHD.7.1.Atmos-FGT.mkv 86.5G
下载到本地后变成了32.8G，使用mediainfo查看确实是和pikpak网页版上使用potplayer播放时选择的4K画质一样

**Comment by NovaShen555** [comment]
不通过alist下载能下载到原大小的文件吗

**Reply by coolyzp** [reply]
> 不通过alist下载能下载到原大小的文件吗

pikpak windows客户端可以

**Comment by xhofe** [comment]
pikpak 因为之前有人提过使用media链接（也就是转码过的）不限制速度，我忘记在哪里提的了，所以在https://github.com/alist-org/alist/commit/8f19c45a817598314a4959362eaae99503bc00f6 之后只要有media链接会优先使用media链接。

**Reply by coolyzp** [reply]
> pikpak 因为之前有人提过使用media链接（也就是转码过的）不限制速度，我忘记在哪里提的了，所以在[8f19c45](https://github.com/alist-org/alist/commit/8f19c45a817598314a4959362eaae99503bc00f6) 之后只要有media链接会优先使用media链接。

有个选项能配置是否使用就好了
